build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -96,11 +96,30 @@ function(iree_bytecode_module)
     get_filename_component(_FRIENDLY_NAME "${_RULE_SRC}" NAME)
   endif()
 
+  # Check LLVM static library setting.
+  foreach(_FLAG ${_RULE_FLAGS})
+    string(FIND
+      "${_FLAG}" "--iree-llvm-static-library-output-path=" _STATIC_OBJ_IDX
+    )
+    if(_STATIC_OBJ_IDX GREATER_EQUAL 0)
+      set(_STATIC_LIB TRUE)
+      string(REPLACE
+        "--iree-llvm-static-library-output-path=" "" _STATIC_OBJ_PATH "${_FLAG}"
+      )
+      string(REPLACE ".o" ".h" _STATIC_HDR_PATH "${_STATIC_OBJ_PATH}")
+    endif()
+  endforeach(_FLAG)
```

Comment:
This is doing a bit more than checking a setting. Can you update the comment? (I'm trying to understand what the string replacements are doing)

Reply:
Comment updated. I also simplify the logic a bit to utilize `REGEX MATCH` to find the object path.